projects/storefrontstyles/scss/components/organization/_organization-home.scss

```diff
@@ -0,0 +1,55 @@
+.CompanyPageTemplate {
+  cx-banner {
+    @extend .col-md-12;
+    @extend .col-lg-5;
+    border-style: solid;
+    border-color: black;
+    border-width: 1px;
+
+    margin: 10px 15px 10px 15px;
+    padding: 10px;
+
+    cx-generic-link {
+      a {
+        position: relative;
+        padding-left: 85px;
+        p:nth-child(1) {
+          text-shadow: none;
+          color: black;
+          padding: 5px;
+          line-height: normal;
+          text-align: left;
+          position: relative;
+          font-size: $font-size-base * 1.375;
+        }
+
+        cx-media {
+          position: absolute;
+          top: 50%;
+          margin-top: -50px;
+          left: -10px;
+          max-height: 100px;
+          max-width: 100px;
+          width: 100%;
+          height: 100%;
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          img {
+            max-width: 60px;
+          }
+        }
+
+        p:nth-child(3) {
+          text-shadow: none;
+          color: black;
+          padding: 5px;
+          line-height: normal;
+          position: relative;
+          text-align: left;
+          font-size: $small-font-size;
+        }
```

Comment:
I know there is no detailed visual specs so it's nothing more than an opinion, but it looks odd to me that the whole description is underlined with red when we hover over it.  Maybe there could be a design iteration to how the whole home page item should behave on hover.

Reply:
agree with this, let me take a look and find out if we can live with no onhover link change. we can allow the merge but do a style update later